ios/Rainbow/Info.plist

```diff
@@ -16,6 +16,15 @@
 	<dict>
 		<key>CFBundleAlternateIcons</key>
 		<dict>
+		  <key>smol</key>
+    	<dict>
+        <key>UIPrerenderedIcon</key>
+        <false/>
+        <key>CFBundleIconFiles</key>
+        <array>
+          <string>smol</string>
+        </array>
+    	</dict>
```

Comment:
Can u fix the indentation here?

Reply:
idk why it's displaying weirdly here but when i view this file locally it is formatted correctly